default_protocol/pipelines.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
I think this pipeline selection as part of configuration should also be placed in nanocrop.